bundle.go

```diff
@@ -15,6 +15,8 @@ import (
 type Bundle interface {
 	// Data returns the contents of the bundle's bundle.yaml file.
 	Data() *BundleData
+	// RawBundle returns the raw string content of a bundle
+	RawBundle() string
```

Comment:
I'm struggling to like this name. Maybe `BundleBytes()`?

Reply:
Yeah that's fair

I'll obviously change the return type to `[]byte` as well